app/controllers/blog_controller.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
 class BlogController < ApplicationController
+  before_action :authenticate_current_user!, except: %i[index]
```

Comment:
By rails style guide only options is more understandable then except: https://www.notion.so/flatstack/Style-Guide-425b94a54f7e481fa3f5dd10b58719ab#b525be45fe7f4ceb801362928f62e4ff

Reply:
Thx!